src/python/gudhi/cover_complex.py

```diff
@@ -154,6 +194,39 @@ def save_to_txt(self, file_name="cover_complex", data_name="data", cover_name="c
                 if len(s) == 2:
                     f.write(str(name2id[s[0]]) + " " + str(name2id[s[1]]) + "\n")
     
+    def save_to_html(self, file_name="cover_complex", data_name="data", cover_name="cover", color_name="color"):
+        """
+        Write the cover complex to an HTML file called "{file_name}.html", that can be visualized in a browser.
+
+        Parameters
+        ----------
+        file_name : string
+            name for the output .txt file, default "cover_complex" 
+        data_name : string
+            name to use for the data on which the cover complex was computed, default "data". It will be used when generating an html visualization with KeplerMapperVisuFromTxtFile.py 
+        cover_name : string
+            name to use for the cover used to compute the cover complex, default "cover". It will be used when generating an html visualization with KeplerMapperVisuFromTxtFile.py
+        color_name : string
+            name to use for the color used to color the cover complex nodes, default "color". It will be used when generating an html visualization with KeplerMapperVisuFromTxtFile.py
```

Comment:
Is KeplerMapperVisuFromTxtFile.py really involved here?

On the other hand, it might be nice to mention that this function uses a fork/clone of the visualisation code of kepler mapper by [this guy].

Reply:
> it might be nice to mention that this function uses a fork/clone of the visualization code of Kepler mapper by [this guy].

Do you think it would be confusing to add that?

Reply:
I think it is better to add it, I'll do this later